### PR TITLE
create quiz home page

### PR DESCRIPTION
**Pull request by git-0r**
# created home page screen for quiz app.
quiz1
quiz2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,10 +6,241 @@
...
+                <div class="quiz-heading-metrics d-flex flex-justify-evenly">
+                    <p class="text-xl">1237 Questions</p>
+                    <p class="text-xl">615 Answers</p>
+                    <p class="text-xl">100K+ Views</p>
+                </div>
+                <button class="btn btn-primary">Take the Quiz</button>
+            </div>
+        </div>
+
+        <p class="heading-lg">Popular Quiz</p>
```
Comment (manish-GH) [inline index.html]: You could probably use a header tag instead of p tag. Do this for other headings as well

File: style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+}
+
+.navbar-input {
+  width: 80%;
+  height: 30px;
```
Comment (manish-GH) [inline style.css]: use rem

File: style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  margin-top: 2rem;
+}
+
+.banner {
+  position: relative;
+}
+
+.banner .banner-img {
+  width: 100%;
+  height: 350px;
```
Comment (manish-GH) [inline style.css]: use rem

File: style.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  object-fit: fill;
+  border-radius: var(--border-radius-lg);
+}
+
+.banner-text {
+  position: absolute;
+  top: 50px;
+  left: 50px;
```
Comment (manish-GH) [inline style.css]: use rem

File: style.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  gap: 2rem;
+}
+
+.newsletter {
+  margin: 5rem auto;
+  flex-direction: column;
+}
+
+.newsletter-email-input {
+  height: 30px;
```
Comment (manish-GH) [inline style.css]: use rem

File: style.css [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  background-color: var(--clr-white-full);
+}
+
+@media screen and (max-width: 600px) {
+  :root {
+    font-size: 66.25%;
+  }
+
+  .banner .banner-img {
+    height: 250px;
```
Comment (manish-GH) [inline style.css]: use rem

File: style.css [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  }
+  .banner-text {
+    top: 20px;
```
Comment (manish-GH) [inline style.css]: use rem

**Review by manish-GH — COMMENTED** [review]
The UI looks clean, it's even responsive. Nice work.

Just change the px to rem wherever possible and consider using the header tags

File: style.css [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+  color: var(--clr-black-full);
+  border-top: 1px solid var(--clr-black-disabled);
+  background-color: var(--clr-primary-hover);
+}
+
+.bg-white-full {
+  background-color: var(--clr-white-full);
+}
+
+@media screen and (max-width: 600px) {
```
Comment (manish-GH) [inline style.css]: You are supposed to use em for media queries

**Review by manish-GH — APPROVED** [review]
I forgot to mention about the em for media queries. Just change that and you are good to merge

**Review by varun1233 — APPROVED** [review]
Code looks good. Ready to merge
Suggestion: It's a good practice to provide alt for images, in some places you are providing it but in some place you aren't. you can look into this

File: index.html [hunk 9]
```diff
@@ -6,10 +6,240 @@
...
+                <path d="M5.52 19c.64-2.2 1.84-3 3.22-3h6.52c1.38 0 2.58.8 3.22 3" />
+                <circle cx="12" cy="10" r="3" />
+                <circle cx="12" cy="12" r="10" />
+            </svg>
+        </div>
+    </nav>
+
+    <main class="main-content">
+        <div class="banner">
+            <img class="banner-img"
```
Comment (Naman-Saxena1) [inline index.html:35]: You can use object-fit: cover on this class., rather than object-fit: fill

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: style.css [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+
+.banner-text {
+  position: absolute;
+  top: 3rem;
+  left: 3rem;
+  color: var(--clr-white-full);
+}
+
+.quiz-container {
+  flex-wrap: wrap;
```
Comment (gouravmarch20) [inline style.css:47]: majority things are find great, flex-wrap: wrap; good to see you had used it

**Review by gouravmarch20 — APPROVED** [review]

**Pull request merged**